relationships.md

```diff
@@ -0,0 +1,160 @@
+    },
+    "rightItem": {
+      "href": "https://dspace7-entities.atmire.com/rest/api/core/items/0ffbee3f-e7ea-42bc-92fe-2fbef1a52c0f"
+    }
+  },
+  "_embedded": {
+    "relationshipType": {
+      "id": 1,
+      "leftLabel": "isAuthorOfPublication",
+      "rightLabel": "isPublicationOfAuthor",
+      "leftMinCardinality": 5,
+      "leftMaxCardinality": 2147483647,
+      "rightMinCardinality": 0,
+      "rightMaxCardinality": 20000,
+      "type": "relationshiptype",
+      "_links": {
+        "leftType": {
+          "href": "https://dspace7-entities.atmire.com/rest/api/core/entitytypes/1"
+        },
+        "rightType": {
+          "href": "https://dspace7-entities.atmire.com/rest/api/core/entitytypes/2"
+        },
+        "self": {
+          "href": "https://dspace7-entities.atmire.com/rest/api/core/relationshiptypes/1"
+        }
+      },
+      "_embedded": {
+        "leftType": {
+          "id": 1,
+          "label": "Publication",
+          "type": "entitytype",
+          "_links": {
+            "self": {
+              "href": "https://dspace7-entities.atmire.com/rest/api/core/entitytypes/1"
+            },
+            "relationshiptypes": {
+              "href": "https://dspace7-entities.atmire.com/rest/api/core/entitytypes/1/relationshiptypes"
+            }
+          }
+        },
+        "rightType": {
+          "id": 2,
+          "label": "Person",
+          "type": "entitytype",
+          "_links": {
+            "self": {
+              "href": "https://dspace7-entities.atmire.com/rest/api/core/entitytypes/2"
+            },
+            "relationshiptypes": {
+              "href": "https://dspace7-entities.atmire.com/rest/api/core/entitytypes/2/relationshiptypes"
+            }
+          }
+        }
+      }
+    }
+  }
+}
+```
+
+The [relationship type](relationshiptypes.md) is embedded
+The 2 items are included as HAL links but are not embedded
+
+## Creating a relationship
+
+**POST /api/core/relationships?relationshipType=<:relationshipType>**
+
+A new relationship between 2 items can be created by specifying both items in a uri-list body and specifying the relationshipType as a parameter
+
+A sample CURL command would be:
+```
+curl -i -X POST 'https://dspace7-entities.atmire.com/rest/api/core/relationships?relationshipType=1' -H 'Authorization: Bearer eyJhbGciO…' -H "Content-Type:text/uri-list" --data 'https://dspace7-entities.atmire.com/rest/api/core/items/12623672-25a9-4df2-ab36-699c4c240c7e \n https://dspace7-entities.atmire.com/rest/api/core/items/5a3f7c7a-d3df-419c-8a2-f00ede62c60a'
+```
+
+The uri-list should always contain exactly 2 items. The first item will be used as the left Item. The second item will be used as the right Item.
+The relationshipType parameter is mandatory as well
+
+## Updating a relationship
+
+**PUT /api/core/relationships/<:id>?relationshipType=<:relationshipType>**
```

Comment:
`PUT` should not have a `relationshipType` param (it's unnecessary).  As noted below, the `relationshipType` cannot be changed (you'd have to DELETE and recreate the relationship)

Reply:
I've updated the parameter, it was indeed decided to be immutable